apps/game_backend/priv/daily_rewards_rules.json

```diff
@@ -0,0 +1,11 @@
+{
+    "daily_rewards_system": {
+        "1": "2",
+        "2": "3",
+        "3": "4",
+        "4": "5",
+        "5": "6",
+        "6": "7",
+        "7": "1"
```

Comment:
What happens after the 7th day?

Reply:
Goes back to first one 😄 
https://www.notion.so/lambdaclass/Daily-Login-9b9db0c820bd49f29b5c75753ad7205b